http/http_client.hpp

```diff
@@ -497,14 +498,19 @@ class ConnectionInfo : public std::enable_shared_from_this<ConnectionInfo>
         }
 
         // Let's close the connection and restart from resolve.
-        doClose(true);
+        doCleanup(true);
+    }
+    void restartConnection()
+    {
+        BMCWEB_LOG_ERROR << "restartConnection ";
```

Comment:
Please enhance this trace by adding the destination ip and port number. And edit as "INFO: restart connection to <host> : <ip>

Reply:
Ack